src/ucs/sys/math.h

```diff
@@ -92,9 +92,9 @@ static inline double ucs_log2(double x)
 
 /**
  * Convert flags without a branch
- * @return 'newflag' oldflag is set in 'value', otherwise - 0
+ * @return '_newflag' if '_oldflag' is set in '_value', otherwise - 0
  */
-#define ucs_convert_flag(value, oldflag, newflag) \
+#define ucs_convert_flag(_value, _oldflag, _newflag) \
```

Comment:
need to update names inside macro as well

Reply:
ops :) done